libs/adapter/file-store/mongo-db.file-store.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { Collection } from 'mongodb';
```

Comment:
Import order

Reply:
Updated, along with segregation of event-store and subscription segregation.